VERSION

```diff
@@ -1 +1 @@
-0.6.2
+0.6.3
```

Comment:
I'd better to not include the version in the PR

Reply:
Normally I create a PR only to change the version and release. This is a quick bug fix release and I was a bit lazy to create another PR. But ok I'll do PRs even for small bug fix releases.

Reply:
no pb, I can approve this. but IMO version should not go into the PR (imagine to cherry picking into another branch). The version can be pushed directly to build the final release